rsmtool/reader.py

```diff
@@ -62,6 +64,11 @@ def __init__(self,
         # Make sure filepaths length matches frame names length
         assert len(filepaths) == len(framenames)
 
+        # Make sure that there are no Nones in the filepaths
+        frames_with_no_path = [framenames[i] for i in range(len(framenames)) if filepaths[i] is None]
+        if len(frames_with_no_path) > 0:
+            raise ValueError("No path specified for {}".format(' ,'.join(frames_with_no_path)))
+
```

Comment:
This is really nit-picky, but I wonder if it's slightly more efficient to just check whether `None` is in the `filepaths` list first, and if so, only then create the `frames_with_no_path` list and raise the `ValueError`. 

```
if None in filepaths:
    frames_with_no_path = [name for idx, name in enumerate(framenames) if filepaths[idx] is None]
    raise ValueError("No path specified for {}".format(' ,'.join(frames_with_no_path)))
```

(This is what you did in `rsmtool.py` below.)